README.md

```diff
@@ -136,14 +136,27 @@ PostDuplicateTagValues - Controls when datapoints are logged.  If set to true, d
 
 CommunicationType - This refers to if you are using OMF (OSIsoft message format). If you have PIWEBAPI 2019 or later, and you have OMF enabled, set this string to "omf". If you are using an older version of PIWEBAPI set the string to "piwebapi". Any other values will result in an error. For more information on OMF read the section here:  [OMF](#OMF-Support)
 
-### TagList
-
-TagList - List of eWON tags that should be connected to the OSIsoft PI server.  If PI Points with (non case sensitive) matching names do not exist on the PI server they will be created automatically.
-
 ## Customizing the application
 
 If you wish to modify, debug, or rebuild the application the toolkit and documentation is available here https://developer.ewon.biz/content/java-0. The instructions for setting up your development environment are here  https://developer.ewon.biz/system/files_force/AUG-072-0-EN-%28JAVA%20J2SE%20Toolkit%20for%20eWON%20Flexy%29.pdf?download=1
 
+## Tag Configuration
+To have a tag's historical data sent to OSIsoft, the tag needs to have historical logging enabled and needs to be assigned to a group. Follow the steps below for each tag.
+
+>Navigate to the tag values page. If the "mode" slider on the top left of the screen is set to "view", click on it to set it to "setup".
+
+![osisoft tag configuration step 1](docs/images/step1.PNG)
+
+>Double click on a tag to open the tag configuration page.
+>Enable historical logging for each tag you want recorded in OSIsoft by clicking the "Historical Logging Enabled" checkbox.
+>For the best application performance, HMS recommends no more than 1 data point be recorded every 10 seconds per tag.
```

Comment:
How was this limit determined? I would think with lower tag counts you could go to once per second.

Reply:
I was thinking of the Nordson limits with large numbers of tags. How about something less specific like: "When using this connector with a large number of tags, HMS recommends a raising the logging period to increase performance."

Reply:
This project does not have the same limitations that Nordon had. I had testing the throughput that that version of the connector was capable of and had a formula for calculating the tag config. 

You are not limited to 1 tag value per tag per message so you will have a very different rate that you need to figure out.

Reply:
There are some issues with the queue library with large amounts of tags and I had a hard time testing this. Exceptions being thrown when getting the next batch of data points at 900 tags from the historical queue lib. I am removing this line for now to get it through the pull request